code/datums/status_effects/buffs.dm

```diff
@@ -178,7 +182,11 @@
 			itemUser.adjustBrainLoss(-1.5)
 			itemUser.adjustCloneLoss(-0.5) //Becasue apparently clone damage is the bastion of all health
 		//Heal all those around you, unbiased
+		if(heal_points < (max_heal_points))
```

Comment:
This comment should probably be moved back to line 186, above the `for()` loop.

Reply:
comment moved